Tests/com/drew/imaging/heif/MetadataExtractorTest.java

```diff
@@ -0,0 +1,249 @@
+package com.drew.imaging.heif;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MetadataExtractorTest {
```

Comment:
Please code alignment in this whole file.

Reply:
Corrected the code alignment